From 9945ad2ac346d36aac34424cef3848ccd8c57a7f Mon Sep 17 00:00:00 2001
From: linmin <linmin@eswincomputing.com>
Date: Wed, 26 Jun 2024 17:51:38 +0800
Subject: [PATCH 096/109] WIN2030-15279:fix(hf_common,bmc2.4):Disable lost
 resume by default

Changelogs:
1.Change the default/initial lost resume attribute to disable.
2.Update bmc version to 2.4

Signed-off-by: linmin <linmin@eswincomputing.com>
Change-Id: I0a1bd3076445fed58d1c512ff55bba879a180ada
---
 Core/Inc/hf_common.h | 2 +-
 Core/Src/hf_common.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Core/Inc/hf_common.h b/Core/Inc/hf_common.h
index 23a88dd..2bcb830 100644
--- a/Core/Inc/hf_common.h
+++ b/Core/Inc/hf_common.h
@@ -242,7 +242,7 @@ extern UART_HandleTypeDef huart3;
 #define MCU_MAC_IDX	2
 /* define ------------------------------------------------------------*/
 #define BMC_SOFTWARE_VERSION_MAJOR                   2
-#define BMC_SOFTWARE_VERSION_MINOR                   3
+#define BMC_SOFTWARE_VERSION_MINOR                   4
 
 #define MAGIC_NUMBER	0x45505EF1
 
diff --git a/Core/Src/hf_common.c b/Core/Src/hf_common.c
index 5f03158..8b0383c 100644
--- a/Core/Src/hf_common.c
+++ b/Core/Src/hf_common.c
@@ -463,7 +463,7 @@ static int get_som_pwrmgt_dip_info(void)
 		skip_update_eeprom = 0;
 		eeprom_debug("Invalid pwr lost resume attr(0x%x) in EEPROM, set to SOM_PWR_LOST_RESUME_ENABLE!\n",
 				gSOM_PwgMgtDIP_Info.som_pwr_lost_resume_attr);
-		gSOM_PwgMgtDIP_Info.som_pwr_lost_resume_attr = SOM_PWR_LOST_RESUME_ENABLE;
+		gSOM_PwgMgtDIP_Info.som_pwr_lost_resume_attr = SOM_PWR_LOST_RESUME_DISABLE;
 	}
 
 	if ((gSOM_PwgMgtDIP_Info.som_pwr_last_state != SOM_PWR_LAST_STATE_ON) &&
-- 
2.25.1

